composer.json

```diff
@@ -46,7 +46,7 @@
         "symfony/phpunit-bridge": "^4.2",
         "symfony/validator": "^3.4|^4.2",
         "symfony/yaml": "^3.4|^4.2",
-        "twig/twig": "^1.12|^2.0"
+        "twig/twig": "~1.34|~2.4"
```

Comment:
Please, stick with `^` for consistency.

Reply:
oops, I copied it from `symfony/framework-bundle`, done!